src/CommonDBTM.php

```diff
@@ -5594,7 +5594,7 @@ public function addFiles(array $input, $options = [])
                     $doc->update($input2);
                 }
             } else {
-                if ($this->getType() == 'Ticket') {
+                if ($this->getType() == 'Ticket' || (isset($input['_job']) && $input['_job'] instanceof Ticket)) {
                     //TRANS: Default document to files attached to tickets : %d is the ticket id
                     $input2["name"] = addslashes(sprintf(__('Document Ticket %d'), $this->getID()));
                     $input2["tickets_id"] = $this->getID();
```

Comment:
`$this->getID()` should probably be replaced by `$input['_job']->getID()` when the ticket is stored in `$input['_job']`.